src/workflows-app/SubmissionHistory.js

```diff
@@ -122,6 +122,22 @@ export const BaseSubmissionHistory = ({ namespace, workspace }, _ref) => {
     }
   };
 
+  const renderInstructionalElements = (userId) => {
+    const introDiv = div(['See workflows that were submitted by all collaborators in this workspace.']);
+    const nicknameDivs = userId
+      ? [
+          div(['Your workspace nickname is ', span({ style: { fontWeight: 'bold' } }, [samIdToWorkspaceNickname(userId)]), '.']),
+          div([
+            'Other users in this workspace will see ',
+            span({ style: { fontWeight: 'bold' } }, [samIdToWorkspaceNickname(userId)]),
```

Comment:
small nit: since `userId` is constant in this method maybe `samIdToWorkspaceNickname(userId)` can be extracted into a variable as it is being called twice here.

Reply:
done!

and I also changed "workspace nickname" to "nickname"